File: Solution/CSharp/27.RemoveElement.cs
public class Solution {
    public int RemoveElement(int[] nums, int val) {
        int index = 0;
        for (int i = 0; i < nums.Length; i++) {
            if (nums[i] == val) {
                length--;
            } else {
                nums[index++] = nums[i];
            }
        }
        return length;
    }
}

File: Solution/Python/27. Remove Element.py
class Solution:
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        length = len(nums)
        index = 0
        for i in range(0, len(nums)):
            if nums[i] == val:
                length -= 1
            else:
                nums[index] = nums[i]
                index += 1
        return length


s = Solution()

nums = [3, 2, 2, 3]
val = 3
res = s.removeElement(nums, val)
print(res)
